ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -466,6 +473,7 @@ const NetworksForm = ({
   ]);
 
   const onSubmit = async () => {
+    setIsEditing(false);
```

Comment:
In case onSubmit ends up failing, should we move this after the updates have been confirmed?